docs/docs/cmd/context/context-remove.md

```diff
@@ -0,0 +1,25 @@
+# context remove
+
+Removes the CLI for Microsoft 365 context in the current working folder
+
+## Usage
+
+```sh
+m365 context remove [options]
+```
+
+## Options
```

Comment:
All remove options should have a `--confirm` option to suppress the confirmation prompt